javascript/cable_ready.js

```diff
@@ -363,12 +363,16 @@ const perform = (
           if (detail.element || options.emitMissingElementWarnings)
             DOMOperations[name](detail)
         } catch (e) {
-          if (detail.element)
-            console.log(`CableReady detected an error in ${name}! ${e.message}`)
-          else
+          if (detail.element) {
+            console.error(
+              `CableReady detected an error in ${name}! ${e.message}. If you need to support older browsers make sure you've included the corresponding polyfills. Check this out for more details: https://docs.stimulusreflex.com/setup#polyfills-for-ie11.`
```

Comment:
I think we can omit this part of the message. `Check this out for more details:`